panel/io/resources.py

```diff
@@ -122,13 +122,8 @@ def conffilter(value):
 
 extension_dirs['panel'] = str(DIST_DIR)
 
-if mimetypes.types_map.get('.js') == 'text/plain':
-    logger.warn(
-        "Mimetype declaration for .js files was set to 'text/plain'. "
-        "This will interefere with serving JS modules. Ensure your system "
-        "does not override the mimetype declaration and/or contact your "
-        "system administrator. "
-    )
+mimetypes.init()
+if mimetypes.types_map.get('.js') != "application/javascript":
```

Comment:
I removed the logger.warn for many reasons including 1) logger.warn is deprecated and 2) it only gives the user problems.

- A newbie would not know what a `Mimetype` declaration or JS Modules is. In my experience they definitely don't like to contact a system administrator.
- An experience, grumpie man like me would have to spend time on explaining this warning to my colleagues. And if I have the energy spend time on reporting this to IT and spend weeks on explaining to them why they have to change the windows registry for 7000 people.

Why warn when everything will work after the code has been executed?

Reply:
Agreed.